nsq/reader.py

```diff
@@ -476,6 +499,7 @@ def connect_to_nsqd(self, host, port):
         conn.on('close', self._on_connection_close)
         conn.on('ready', self._on_connection_ready)
         conn.on('message', self._on_message)
+        conn.on('message_timeout', self._on_message_timeout)
```

Comment:
Added [a test][1] to confirm

[1]: https://github.com/akellehe/pynsq/blob/message-timeout-callback-and-state-accessor/tests/test_reader.py#L201-L223